src/py42/response.py

```diff
@@ -111,7 +111,10 @@ def _data_root(self):
             if not self._data:
                 response_dict = json.loads(self._response.text)
                 if type(response_dict) == dict:
-                    self._data = response_dict.get("data") or response_dict
+                    if "data" in response_dict:
+                        self._data = response_dict["data"]
+                    else:
+                        self._data = response_dict
```

Comment:
So if there's no `"data"` key, we just set the property to the entire response dictionary?

Reply:
Yep, it'a a convenience feature, as a lot of APIs return results like: 
```{"metadata": ..., "data": "the_stuff_you_want"}```
so if there is a "data" field we make it directly accessible on the Response.data property, you can do `response["thing_I_want"]` instead of having to do `response["data"]["thing_I_want"]` every time.